src/components/recipe-timing-information.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import titleize from 'titleize';
+
+import { formatMinutes } from '../utils/recipe-timing-information';
+import styles from './recipe-timing-information.module.css';
+
+const renderTimingInformationCell = cell => (
+  <div key={cell.name} className={styles.cell}>
+    <div className={styles.cellText}>{titleize(`${cell.name} time`)}</div>
+    <div className={styles.cellText}>{formatMinutes(cell.value)}</div>
+  </div>
+);
+
+const renderTimingInformationCells = cells => {
+  const renderedCells = cells.map(renderTimingInformationCell);
+
+  if (cells.length > 1) {
+    const totalTime = cells.map(({ value }) => value).reduce((prev, curr) => prev + curr, 0);
```

Comment:
You can avoid the extra array by just reducing over cells:

```js
cells.reduce((accumulatedTime, { value: currentTime }) => accumulatedTime + currentTime, 0);
```

Reply:
Yup, that is cleaner